content/kubermatic/master/tutorials_howtos/how_to_contribute_to_kkp/_index.en.md

```diff
@@ -0,0 +1,63 @@
+
+**Contributor's Notes**
+
+There are few things to note when contributing to the KKP project, which is highlighted below:
+
+
+
+*   KKP project is hosted on Github; thus, Github knowledge is one of the essential pre-requisites 
+*   The KKP documentation is written in markdown language (.md) and located in the [docs repository](https://github.com/kubermatic/docs/tree/master/content/kubermatic)
+*   See [CONTRIBUTING.md](https://github.com/kubermatic/kubermatic/blob/master/CONTRIBUTING.md) for instructions on the developer certificate of origin that we require
+*   Familiarization with Hugo for building static site locally is suggested for documentation contribution
+*   Kubernetes knowledge is also recommended
+*   The KKP documentation is currently available only in the English Language  
+*   We have a simple code of conduct that should be adhered to
+
+    **Steps in Contributing to KKP**
+
+*   Please familiarise yourself with our [Code of Conduct](https://github.com/kubermatic/kubermatic/blob/master/CODE_OF_CONDUCT.md)
+*   Check the [opened issues](https://github.com/kubermatic/kubermatic/issues) on our GitHub repo peradventure there might be anyone that will be of interest
+*   Fork the repository on GitHub
+*   Read the [README](https://github.com/kubermatic/kubermatic/blob/master/README.md) for build and test instructions
+
+    **Contribution Workflow**
+
+
+The below outlines show an example of what a contributor's workflow looks like:
+
+
+
+*   Fork the repository on GitHub
+*   Create a topic branch from where you want to base your work (usually master)
+*   Make commits of logical units.
+*   Make sure your commit messages are in the proper format
+*   Push your changes to the topic branch in your fork repository
+*   Make sure the tests pass and add any new tests as appropriate
+*   Submit a pull request to the original repository
+*   Assign a reviewer if you wish and wait for the PR to be reviewed 
+*   If everything works fine, your PR will be merged into the project's master branch
+
+Congratulations!!! You have successfully contributed to the KKP project. 
+
+**Communication Channels**
+
+If you encounter a problem, you can [file an issue](https://github.com/kubermatic/kubermatic/issues) in the repository on GitHub or talk to us on [GitHub](https://github.com/kubermatic/kubermatic/discussions), [#kubermatic slack channel](https://kubermatic.slack.com/messages/kubermatic), and also through our [forum link](https://forum.kubermatic.com/c/kubermatic/6). In case of complex issues, we can set up video meetings for resolution. 
```

Comment:
> In case of complex issues, we can set up video meetings for resolution.

That's quite a promise to random contributors. Who volunteered to be the video person? :-D

Reply:
@sascha